modules/gathering-data-specific-features.adoc

```diff
@@ -60,15 +60,12 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:v<installed_version_sandboxed_containers>`
 |Data collection for {sandboxed-containers-first}.
 
-|`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:v<installed-version-SNR>`
-|Data collection for the Self Node Remediation (SNR) Operator and the Node Health Check (NHC) Operator.
+|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
+|Data collection for the Red Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
```

Comment:
s/Red Hat/Red{nbsp}Hat

Reply:
Thanks @StephenJamesSmith for your feedback. I have applied your suggestion.